about-page.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en-us">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Go Green - Summit 2022</title>
+    <meta name="description" content="About Go Green - Summit 2022">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="stylesheets/styles.css">
+</head>
+
+<body class="about">
+    <section class="top-section">
+        <div class="top-cover">
+            <nav class="inactive">
+                <a href="#close"><img src="images/bars.png" alt="Nav Button"></a>
+                <a href="index.html" class="logo"><img src="images/green_logo.png" alt="logo"></a>
+                <ul>
+                    <li class="Home">
+                        <a href="index.html">Home</a>
+                    </li>
+
+                    <li>
+                        <a href="about-page.html" class="active">About</a>
+                    </li>
+                    
+                    <li>
+                        <a href="#">Program</a>
+                    </li>
+                    
+                    <li>
+                        <a href="#">Join</a>
+                    </li>
+                    
+                    <li>
+                        <a href="#">Sponser</a>
+                    </li>
+                    
+                    <li>
+                        <a href="#">News</a>
+                    </li>
+                    
+                    <li>
+                        <a href="#">CC Compaign</a>
+                    </li>
+                </ul>
+            </nav>
+            <div class="content">
+                <header>
+                    <p>"Hello! Sharing world"</p>
+                    <h1>GoGreen Summit 2022</h1>
+                </header>
+                <p>
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptas earum quia eos ad modi consequuntur! Reprehenderit cupiditate, maxime.
+                </p>
+                <div class="top-section-desc">
+                    <p>Please contact us per Email for any further questions about GoGreen Summit 2022</p>
+                    <a href="mailto:saadatali0202@gmail.com">gogreen@gmail.com</a>
+                </div>
+            </div>
```

Comment:
- [ ] I'd like you to notice that the medium-sized text in this section should be wrapped inside a container with a border. If you look closely at the reference design you will see this. please kindly implement this border as per the reference design

| Your Design| Reference Design |
-|-
|![image](https://user-images.githubusercontent.com/18730106/152666346-ca289563-4b52-4643-9cb2-2d496dce2844.png)|![image](https://user-images.githubusercontent.com/18730106/152666357-e2fdd0bf-800e-4cc8-bf46-c4b14fcffb77.png)|

- [ ] the bottom text with a contact email should be bold as per the reference design

|Your Design|Reference Design|
-|-
|![image](https://user-images.githubusercontent.com/18730106/152666372-7b19cfbc-5854-4645-94bc-8b8b5e8137d7.png)|![image](https://user-images.githubusercontent.com/18730106/152666379-447f5224-aa90-40f7-9b7b-a91d15c77b06.png)|